subgraph/src/maci.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-underscore-dangle */
+import { Address, BigInt as GraphBN } from "@graphprotocol/graph-ts";
+
+import {
+  DeployPoll as DeployPollEvent,
+  OwnershipTransferred as OwnershipTransferredEvent,
+  SignUp as SignUpEvent,
+} from "../generated/MACI/MACI";
+import { Poll } from "../generated/schema";
+import { Poll as PollTemplate } from "../generated/templates";
+import { Poll as PollContract } from "../generated/templates/Poll/Poll";
+
+import { ONE_BIGINT } from "./utils/constants";
+import { createOrLoadMACI, createOrLoadUser, createOrLoadAccount } from "./utils/helper";
+
+export function handleDeployPoll(event: DeployPollEvent): void {
+  const maci = createOrLoadMACI(event);
+
+  const entity = new Poll(event.params.pollAddr.poll);
+  const contract = PollContract.bind(event.params.pollAddr.poll);
+  const maxValues = contract.maxValues();
+  const treeDepths = contract.treeDepths();
+  const durations = contract.getDeployTimeAndDuration();
+
+  entity.pollId = event.params._pollId;
+  entity.messageProcessor = event.params.pollAddr.messageProcessor;
+  entity.tally = event.params.pollAddr.tally;
+  entity.subsidy = event.params.pollAddr.subsidy;
+  entity.maxMessages = maxValues.value0;
+  entity.maxVoteOption = maxValues.value1;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Why do you need this disabled rule here?

Reply:
I could not figuring out a way to overcome this...

Reply:
I could not figuring out a way to overcome this...

Reply:
So, if it was something related to eslint, it will be a problem for maxValues as well, so seems it's a type mismatch error. You need to check `treeDepths` and and `treeDepth` for entity.